examples/asr/conf/asr_adapters/asr_adaptation.yaml

```diff
@@ -57,10 +57,12 @@ model:
   pretrained_model: null  # name of a pretrained model
   nemo_model: null  # path to a ASR model file (.nemo)
 
+  log_prediction: false # enables logging sample predictions in the output during training
+
   adapter:
     # Config of the adapter training/eval script.
     adapter_name: ???  # Name of the adapter, used by the script
-    adapter_module_name: null  # Name of the preprocessing module, if required. Defaults to None, equivalent to ''.
+    adapter_module_name: null  # Name of the adapter module. Combine multiple modules with '+' between module names.
```

Comment:
Are we going to have more detail about this new support in the docs or tutorials?